.woodpecker/.latest.yml

```
@@ -0,0 +1,10 @@
+steps:
+  build:
+    image: woodpeckerci/plugin-docker-buildx
+    settings:
+      repo: ${CI_REPO}
+      tags: latest
+    secrets: [ docker_username, docker_password ]
+when:
+  branch: master
```

It seems this was set to 'development' in the drone version, I would keep it to make the behavior consistent.
```suggestion
  branch: development
```